File: Onion.Domain/Entities/Pedido.cs
namespace Onion.Domain.Entities;

public sealed class Pedido : Entity
{
    public int Numero { get; private set; }
    public string CEP { get; private set; } = string.Empty;
    public int ClienteId { get; private set; }
    public Cliente Cliente { get; private set; } = new();
    public Produto Produto { get; private set; } = new();
    public DateTime DataCriacao { get; private set; }
}

File: onion-app/src/types/pedido.ts
import { Cliente } from "./cliente"
import { Produto } from "./produto"

export type Pedido = {
	id: number
	numeroPedido: number
	cep: string
	uf: string
	valorFinal: number
	produto: Produto
	cliente: Cliente
	dataCriacao: string
	dataEntrega: string
}
